common/persistence/persistence-tests/clusterMetadataManagerTest.go

```diff
@@ -120,17 +128,23 @@ func (s *ClusterMetadataManagerSuite) TestClusterMembershipUpsertCanPageRead() {
 		resp, err := s.ClusterMetadataManager.GetClusterMembers(&p.GetClusterMembersRequest{PageSize: 9, NextPageToken: nextPageToken})
 		s.NoError(err)
 		nextPageToken = resp.NextPageToken
-		hostCount += len(resp.ActiveMembers)
+		for _, member := range resp.ActiveMembers {
+			expectedIds[primitives.UUIDString(member.HostID)]--
+			hostCount++
+		}
 
 		if nextPageToken == nil {
 			break
 		}
 	}
 
-	s.Equal(hostCount, 100)
+	s.Equal(100, hostCount)
```

Comment:
[Nit]. You can also just create another actual map (actualIds), populate that using ++ and then use s.Equal to compare with expectedIds